src/views/CHConfigEditor.tsx

```diff
@@ -117,23 +130,10 @@ export const ConfigEditor: React.FC<Props> = (props) => {
             onChange={(e) => onPortChange(e.currentTarget.value)}
             label={Components.ConfigEditor.ServerPort.label}
             aria-label={Components.ConfigEditor.ServerPort.label}
-            placeholder={Components.ConfigEditor.ServerPort.placeholder}
+            placeholder={Components.ConfigEditor.ServerPort.placeholder(jsonData.secure?.toString() || 'false')}
```

Comment:
not really satisfied with this but didn't wanna extend the E2E-Selector package just for this